### DISCUSSION TITLE
[Feature] Folder Structure

### DISCUSSION THREAD

**Discussion by kinetic-it** [immich-app/immich#3839] [Feature Request]
### The feature

Is it planned in the near future that you can use folders as well as albums?

I have almost 10 years of pictures here, sorting them all into albums would take years.
Therefore my question if I can upload the folders somehow.
From me it will be albums (reluctantly but possible).

Only there must be a way to maintain its existing structure.

Greetings

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #1006
